bin/impl/setup-accumulo.sh

```diff
@@ -48,6 +48,10 @@ else
   "$ACCUMULO_HOME"/bin/accumulo-cluster create-config
   $SED "s#localhost#$UNO_HOST#" "$conf/tservers"
   cp "$UNO_HOME"/conf/accumulo/accumulo-site-2.0.xml "$conf"/accumulo-site.xml
+  $SED "s#instance.name=#instance.name=$ACCUMULO_INSTANCE#" "$conf"/accumulo-client.properties
```

Comment:
ok. does `instance\.name` work too?

Reply:
If you escape the escape character properly in the double-quoted string, it will work... but escaping the escape to escape the dot, is a bit confusing to me... so I tend to stick with the simple character-class using the square braces. It makes the regex more "portable" between different parsers and languages, and String quoting mechanisms.